app/controllers/internal_controller.rb

```diff
@@ -16,15 +17,17 @@ def set_account_role(role_type)
 
   def authenticate
     authenticate_or_request_with_http_basic do |username, password|
-      return false if username.blank? || password.blank?
```

Comment:
How come we're losing this check? I think its a good idea to keep it in case the ENV vars are not set to avoid allowing a blank username/password pass authentication?

Reply:
The method always returns false if the password and username don't match the environment variables

Reply:
Oh I see, I'll put it back in

Reply:
But what if the environment variables are not set/therefore are `nil`/blank? would it match username `""` and password `""`?